web_widget_uom/readme/USAGE.rst

```diff
@@ -0,0 +1,40 @@
+In the UoM form view specify the number of decimal places that
+should be displayed for each unit of Measure:
+
+.. image:: ../static/description/UoMDecimalPlaces.png
+   :alt: Decimal Places in UoM form view
+
+In the view declaration, put widget='uom' attribute in the field tag::
+
+    ...
+    <field name="arch" type="xml">
+        <tree string="View name">
+            ...
+            <field name="" widget="uom"/>
+            ...
+        </tree>
+    </field>
+    ...
+    <field name="arch" type="xml">
+        <form string="View name">
+            ...
+            <field name="" widget="uom"/>
+            ...
+        </form>
+    </field>
+    ...
+
+Widget Options::
+
+    ...
+    <field name="arch" type="xml">
+        <tree string="View name">
+            ...
+            <field name="color" widget="color" options="{'uom_field': 'product_uom'}"/>
```

Comment:
I know how it works, but this can be confusing for other devs. (`name="color" widget="color"`)

Reply:
done